cpp/src/arrow/csv/options.h

```diff
@@ -119,6 +119,13 @@ struct ARROW_EXPORT ReadOptions {
 
   /// Whether to use the global CPU thread pool
   bool use_threads = true;
+  /// TODO: This will come out before this gets merged in.  There is not much point in
```

Comment:
Ready to remove this?

Reply:
Done.  I went ahead and just removed the threaded table reader entirely.  Now that I've done more benchmarking I don't see any reason for keeping it in.